gensim/utils.py

```diff
@@ -266,6 +266,10 @@ def load(cls, fname, mmap=None):
         obj = unpickle(fname)
         obj._load_specials(fname, mmap, compress, subname)
         logger.info("loaded %s", fname)
+        if cls.__name__ is 'Similarity':
```

Comment:
Hm, I haven't though of that.

So there's some sort of circular dependency? The current solution may be the simplest then (just with `==` instead of `is`).